docs/style.md

```diff
@@ -15,7 +15,9 @@
 * Strings in worlds should use double quotes as well, but imported code may differ.
 * Prefer [format string literals](https://peps.python.org/pep-0498/) over string concatenation,
   use single quotes inside them: `f"Like {dct['key']}"`
-* Use type annotation where possible.
+* Use type annotations where possible for function signatures and class members.
+* Use type annotations where appropriate for local variables (e.g. `var: List[type] = []`, or when the
+  type is hard or impossible to deduce.) Clear annotations help developers look up and validate API calls.
```

Comment:
Also minor: There are 2 bullet points, but the sentence at the end of the 2nd seems more relevant to the 1st than the 2nd.
I'm not sure whether it would disrupt the flow if moved.

Reply:
It looks worse if it's in between

Reply:
Personally, I would just remove that last sentence.